bentoml/_internal/configuration/containers.py

```diff
@@ -89,6 +89,15 @@ def _is_ip_address(addr: str) -> bool:
             "workers": Or(And(int, _larger_than_zero), None),
             "timeout": And(int, _larger_than_zero),
             "max_request_size": And(int, _larger_than_zero),
+            Optional("ssl"): {
+                Optional("keyfile"): Or(str, None),
+                Optional("certfile"): Or(str, None),
+                Optional("keyfile_password"): Or(str, None),
+                Optional("version"): Or(And(int, _larger_than_zero), None),
+                Optional("cert_reqs"): Or(int, None),
+                Optional("ca_certs"): Or(str, None),
+                Optional("ciphers"): Or(str, None),
+            },
```

Comment:
I think this is great. You can also add these options to `BentoMLContainerClass` below.

Reply:
There's no need to, I think?

Reply:
there's no need to add these to BentoMLContainerClass